packages/block-library/src/template-part/edit/get-tag-based-on-area.js

```diff
@@ -0,0 +1,9 @@
+const AREA_TAGS = {
+	footer: 'footer',
+	header: 'header',
+	unactegorized: 'div',
```

Comment:
Typo :grimacing: 

```suggestion
	uncategorized: 'div',
```

Reply:
Or maybe we should consider renaming to something else, now that we're using "area" rather than "category.

Maybe `unassigned`? :thinking:

Reply:
Some template parts need not be assigned to a specific area, but "uncategorized" or "unassigned" might make them feel miss-placed or unorganised. That doesn't feel like the appropriate tone. 

Perhaps something like "General" or "Universal"?

Reply:
"uncategorized" or "unassigned" seems appropriate in the case that an item has been created and this value has never been assigned or distinguished.  Or that no other existing value settings are appropriate.

"General" and "Universal" could definitely be 'areas', but they do not fit the same purpose as uncategorized/unassigned.  We cannot assume something is "Universal" just because it doesn't fit into the existing categories, and when an item is not universal and does not fit into existing categories something like unassigned/uncategorized serves a purpose.

Reply:
https://github.com/WordPress/gutenberg/pull/29937 to fix the typos.

> Perhaps something like "General" or "Universal"?

I just realized/remembered on the user facing side, 'uncategorized' is being displayed as "General".